projects/utils-maps/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@dlr-eoc/utils-maps",
+  "main": "src/public-api",
+  "version": "0.0.0-PLACEHOLDER",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+  },
+  "dependencies": {
```

Comment:
Here we need ol as a dependency, because we us it in the Package see e.g. https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/projects/map-ol/package.json.

Also we use [zone.js and @angular/core](https://github.com/dlr-eoc/ukis-frontend-libraries/pull/38/files#diff-25286687f532aa8ac1bc058a4500276b) for development so it should be listed in **devDependencies**